include/graphblas/reference/vector.hpp

```diff
@@ -855,6 +855,24 @@ namespace grb {
 #endif
 			}
 
+			/**
+			 * Creates a reference vector with given values.
+			 * This constructor takes an initializer list of values that will be copied.
+			 */
+			Vector( const std::initializer_list< D > vals )
+				: Vector( vals.size(), vals.size() )
+			{
+#ifdef _DEBUG
+				std::cerr << "In Vector< reference >::Vector( initializer_list )"
+					<< " constructor\n";
+#endif
+				size_t i = 0;
+				for( auto each : vals ) {
```

Comment:
This actually should be a `for( size_t i = 0; i < vals.size(); ++i )` since the parallel version, guarded by a `_H_GRB_REFERENCE_OMP_VECTOR`, should define a pragma calling for parallel execution using an OpenMP static schedule.